src/main/java/org/sonatype/nexus/plugins/conda/internal/util/CondaPathUtils.java

```diff
@@ -34,6 +34,14 @@
 
   public static final String CHANNELDATA_JSON = "channeldata.json";
 
+  public static final String REPODATA_JSON = "repodata.json";
+
+  public static final String REPODATA_JSON_BZ2 = "repodata.json.bz2";
+
+  public static final String REPODATA2_JSON = "repodata2.json";
+
+  public static final String TAR_BZ2 = ".tar.bz2";
```

Comment:
I was on the fence of just doing `tar.bz2` here, and keeping the period in the places I use it, because it makes the String.format stuff read better. Any opinions?

Reply:
No strong opinion either way - how you have it is fine by me.
I would feel better if we had some test coverage of the other place where this const is used: org.sonatype.nexus.plugins.conda.internal.CondaRecipeSupport#archCondaPackageMatcher, but I think that may better wait until more UT's/IT's come along...